generators/lib/config/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Generates the configuration file from user answers.
+ *
+ * @param {string} subDomain
+ * @param {string} botUsername
+ * @param {string} botEmailAddress
+ * @param {object} rsaConfig
+ * @param {object} certConfig
+ * @returns {{}}
+ */
+module.exports.generateConfig = function (subDomain, botUsername, botEmailAddress, rsaConfig, certConfig) {
+
+    const host = subDomain + '.symphony.com';
+
+    const config = {
+
+        sessionAuthHost: host,
```

Comment:
is it possible to have a conf file and upload it to initialize this default values?